cypress/support/step_definition/course/commonCourseSteps.spec.js

```diff
@@ -201,9 +201,12 @@ When("I click on Delete in dot menu", () => {
 	courses.clickDeleteInDotMenu();
 });
 
-When("I click on Copy in dot menu", () => {
-	courses.clickCopyOptionInThreeDotMenuOfTopic();
-});
+When(
+	"I select the three dot menu action for task on course detail {string}",
```

Comment:
here we are making this step dynamic but calling the copy option method instead

Reply:
this step is now updated with indexing, as in the above comment.